public/js_v2.0.1_f1/stream_functions.js

```diff
@@ -51,9 +51,16 @@ function buildRequestObject(msgAttributes, onData) {
     });
 }
 
-function streamContentreceived(){
+function ContentReceived(data) {
```

Comment:
Let's stick with consistent lowercase first function naming, i.e. this should be `contentReceived`